usr.sbin/adduser/adduser.sh

```diff
@@ -202,6 +203,8 @@ save_config() {
 	echo "msgfile=$msgfile"		>> ${ADDUSERCONF}
 	echo "disableflag=$disableflag" >> ${ADDUSERCONF}
 	echo "uidstart=$uidstart"       >> ${ADDUSERCONF}
+	echo "Zflag=$Zflag"       >> ${ADDUSERCONF}
+	echo "Zencrypt=$Zencrypt"       >> ${ADDUSERCONF}
```

Comment:
I don't see a manpage addition for `Zencrypt`, but with a more descriptive name for `Zflag` you could make that instead a tri-state: `zfscreate=yes|no|encrypted`

Reply:
You're right, Zflag is super confusing!  I set Zcreate and Zencrypt to yes/no bools but didn't implement the tri-state since the two booleans seem to "flow" better in the script.